common/src/main/java/net/william278/huskclaims/config/Settings.java

```diff
@@ -281,6 +281,10 @@ public static class BanSettings {
                     "Also requires the MANAGE_BANS privilege (by default, restricted to those with 'manage' trust)"})
             private boolean enabled = false;
 
+            @Comment({"Whether to let users set their claim claims to private (prevent others from entering) using /claimprivate",
+                    "Also requires the MANAGE_BANS privilege (by default, restricted to those with 'manage' trust)"})
+            private boolean private_claims = false;
```

Comment:
Let's name this variable in camel case - privateClaims

Reply:
👍